install.sh

```diff
@@ -163,20 +150,13 @@ if [ ! -f device.json ]; then
     '. | .[$key0]=$value0 | .[$key1]=[]' <<<'{}' > device.json
 fi
 
-for package in $BOOTSTRAP_PACKAGES; do
-  pkgfile="${CREW_PACKAGES_PATH}/${package}.rb"
-
-  [[ "$(sed -n '/binary_sha256/,/}/p' "${pkgfile}")" =~ .*${ARCH}:[[:blank:]]*[\'\"]([^\'\"]*) ]]
-    sha256s+=("${BASH_REMATCH[1]}")
+# Functions to maintain packages
 
-  [[ "$(sed -n '/binary_url/,/}/p' "${pkgfile}")" =~ .*${ARCH}:[[:blank:]]*[\'\"]([^\'\"]*) ]]
-    urls+=("${BASH_REMATCH[1]}")
-done
-
-# functions to maintain packages
 function download_check () {
+    mkdir -p "$CREW_CACHE_DIR"
+    sudo chown -R "$(id -u)":"$(id -g)" "$CREW_CACHE_DIR"
```

Comment:
These lines should be inside the `if [ -n "$CREW_CACHE_ENABLED" ] ...` conditional.